js/src/views/edit-attribute-field-attachment.js

```diff
@@ -10,11 +18,10 @@ var editAttributeFieldAttachment = sui.views.editAttributeField.extend( {
 	},
 
 	/**
-	 * Update the field attachment.
-	 * Re-renders UI.
-	 * If ID is empty - does nothing.
+	 * Update the Field Attachment to use a new attachment ID.
+	 * Re-renders UI. But, bails early if the ID isn't set.
 	 *
-	 * @param {int} id Attachment ID
+	 * @param integer id Attachment ID
```

Comment:
What is the distinction between when a type description is shown in braces and not? Should this be `{integer}`?